src/EnergyPlus/PlantLoopEquip.cc

```diff
@@ -308,19 +307,19 @@ namespace PlantLoopEquip {
 		//Pipe has no special types at the moment, so find it this way
 		if ( GeneralEquipType == GenEquipTypes_Pipe ) {
 			if ( EquipTypeNum == TypeOf_Pipe ) {
```

Comment:
Maybe collapsing all of the types that use `->simulate` into a single `if` clause would be "better":

    if ( EquipTypeNum == TypeOf_Pipe  || EquipTypeNum == TypeOf_PipeSteam || EquipTypeNum == TypeOf_PipeExterior ...) {